runtime/domain.c

```diff
@@ -557,7 +560,7 @@ static void create_domain(uintnat initial_minor_heap_wsize) {
   domain_state->intern_state = NULL;
 
   domain_state->current_stack =
-      caml_alloc_main_stack(Stack_size / sizeof(value));
+      caml_alloc_main_stack(stack_size / sizeof(value));
```

Comment:
I don't understand the `/ sizeof(value)` here.

For example, suppose `caml_max_stack_size` is a slack constraint and we would be trying to do `Stack_size / sizeof(value)` as the old code did. This new code will set the variable to:
  `Wsize_bsize(stack_size) / sizeof(value)`
that is:
  `(Stack_size / sizeof(value)) / sizeof(value)`

 I make the new variable `stack_size` in units of words, maybe it should be called `stack_wsz`? Or have I got confused somehow?

Reply:
right, this is a terrible mistake. let me fix this.